### ISSUE TITLE
rtmidi installation issue?

### ISSUE THREAD

**Issue by dlamon1** [mido/mido#171]
This code, using Python v3.6

```
import mido

port = mido.open_output('Bus 1')

cue = input("cue?")
while cue != "":
    cue = input() 
    if cue == "1":
        msg = mido.Message('note_on', note=0, velocity=64, time=1)
        port.send(msg)
        print("1")
    elif cue == "2":
        print("2")
    elif cue == "3":
        print("3")

```

Is giving me these errors

> Traceback (most recent call last):
>   File "/Users/devonlamond/PycharmProjects/test/midiNEW.py", line 3, in <module>
>     port = mido.open_output('Bus 1')
>   File "/Users/devonlamond/PycharmProjects/test/venv/lib/python3.6/site-packages/mido/backends/backend.py", line 111, in open_output
>     return self.module.Output(name, **self._add_api(kwargs))
>   File "/Users/devonlamond/PycharmProjects/test/venv/lib/python3.6/site-packages/mido/backends/backend.py", line 41, in module
>     self.load()
>   File "/Users/devonlamond/PycharmProjects/test/venv/lib/python3.6/site-packages/mido/backends/backend.py", line 57, in load
>     self._module = importlib.import_module(self.name)
>   File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/importlib/__init__.py", line 126, in import_module
>     return _bootstrap._gcd_import(name[level:], package, level)
>   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
>   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
>   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
>   File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
>   File "<frozen importlib._bootstrap_external>", line 678, in exec_module
>   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
>   File "/Users/devonlamond/PycharmProjects/test/venv/lib/python3.6/site-packages/mido/backends/rtmidi.py", line 8, in <module>
>     import rtmidi
> ModuleNotFoundError: No module named 'rtmidi'
>

**Comment by dlamon1** [comment]
I tried running 

`pip3 install --user --pre python-rtmidi`

from another thread with a similar error to no avail

**Comment by fornof** [comment]
Are you on a mac? If so, you probably got an error similar to this: 

> _CORE__ -Isrc/rtmidi -I/Library/Frameworks/Python.framework/Versions/3.5/include/python3.5m -c src/_rtmidi.cpp -o build/temp.macosx-10.6-intel-3.5/src/_rtmidi.o -frtti
>   warning: include path for stdlibc++ headers not found; pass '-std=libc++' on the command line to use the libc++ standard library instead [-Wstdlibcxx-not-found]
>   src/_rtmidi.cpp:643:10: fatal error: 'ios' file not found
>   #include "ios"
>            ^~~~~
>   1 warning and 1 error generated.
>   error: command '/usr/bin/clang' failed with exit status 1
The problem occurs when the compiler tries to use a library that has been replaced in later xcode versions and is no longer included.
 You will need to install xcode command line tools : 
`xcode-select --install`

and export your environment variables : 
```
export MACOSX_DEPLOYMENT_TARGET=10.10 ;export CFLAGS='-std=c++11 -stdlib=libc++
t
```
then try 
pip3 install python-rtmidi

**Comment by whyboris** [comment]
🙇‍♂️ thank you for above ❤️ 

TL;DR: 

```sh
pip install python-rtmidi
```
